requirements.txt

```diff
@@ -31,14 +31,14 @@ pyramid-tm==2.2.1
 pyramid==1.10.1
 python-dateutil==2.6.1    # via alembic
 python-editor==1.0.3      # via alembic
-raven==6.9.0
 requests-oauthlib==1.0.0
 requests==2.20.1
+sentry-sdk==0.5.4
 six==1.11.0               # via pylti, python-dateutil
 sqlalchemy==1.2.14
 transaction==2.1.2        # via pyramid-tm, zope.sqlalchemy
 translationstring==1.3    # via pyramid
-urllib3==1.22             # via requests
+urllib3==1.22             # via requests, sentry-sdk
```

Comment:
There doesn't seem to be a way to run `pip-compile` on the new `requirements.in` and just add the new `sentry-sdk` package without also upgrading other packages in the process.

Reply:
Hmm, rebase must have gotten rid of the other changes